standalone/rewrite_config.go

```diff
@@ -91,8 +91,8 @@ func main() {
 		path := home_path + "/testbed/testbeds/default/" + local_node_id + "/config"
 
 		//allow connections from external nodes
```

Comment:
can you delete the comment. since it's loopback address now.

Reply:
thanks, good catch.